picard/file.py

```diff
@@ -106,6 +108,7 @@ class FileErrorType(Enum):
     UNKNOWN = auto()
     NOTFOUND = auto()
     NOACCESS = auto()
+    MUTAGEN = auto()
```

Comment:
What do you think about naming this more generically `PARSER`?

Reply:
> What do you think about naming this more generically `PARSER`?

Yes, better, done in https://github.com/metabrainz/picard/pull/2245/commits/35b04f5bccf15c10e070195e2314e54d53187877